helper-cli/src/main/kotlin/commands/CreateAnalyzerResultCommand.kt

```diff
@@ -227,7 +227,6 @@ private fun ScannedPackage.toPackage(): Package {
     return Package(
         id = id,
         declaredLicenses = emptySet(),
-        concludedLicense = null,
```

Comment:
Note that alternatively, you could have sticked with using `Package.EMPTY.copy()`, but also pass `purl = id.toPurl()` to it.